app/Http/Controllers/Staff/RssController.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D is open-sourced software licensed under the GNU General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D
+ *
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ * @author     singularity43
+ */
+
+namespace App\Http\Controllers\Staff;
+
+use Illuminate\Http\Request;
+use App\Rss;
+use App\Category;
+use App\Type;
+use Brian2694\Toastr\Toastr;
+use App\Repositories\TorrentFacetedRepository;
+use App\Http\Controllers\Controller;
+
+class RssController extends Controller
+{
+    /**
+     * @var TorrentFacetedRepository
+     */
+    private $torrent_faceted;
+
+    /**
+     * @var Toastr
+     */
+    private $toastr;
+
+    /**
+     * RssController Constructor
+     *
+     * @param TorrentFacetedRepository $torrent_faceted
+     * @param Toastr $toastr
+     */
+    public function __construct(TorrentFacetedRepository $torrent_faceted, Toastr $toastr)
+    {
+        $this->torrent_faceted = $torrent_faceted;
+        $this->toastr = $toastr;
+    }
+
+    /**
+     * @param Toastr $toastr
+     */
+    public function setToastr($toastr)
```

Comment:
what is point of this?

Reply:
Probably a hangover from when I was still thinking of how to approach this.

Nuke.

Reply:
Cool I am going to merge both PRs. Then run the code through StyleCI since it doesn't follow UNIT3D code flow. Then do alittle manual cleanup.

Reply:
Sounds good. Let me know what I can do to fix StyleCl errors in future.

About manual cleanups, please discord me any major issues. I left some notes in what I knew was wrong.  Will keep in mind for future.

Reply:
Will do. Its nothing major. Just want to keep things standardized with what UNIT3D already does. Like `$rss = $user->rss()->where('is_private',0)->findOrFail($id);` to `$rss = $user->rss()->where('is_private', '=', 0)->findOrFail($id);`. I know there the same but we decided to show `'='` for clarity to noobs. 

`$this->toastr->error($error, 'Uhoh!', ['options']));` to `$this->toastr->error($error, 'Whoops!', ['options']));`

`$this->toastr->success($success, 'Congratulations!', ['options']));` to `$this->toastr->success($success, 'Yay!', ['options']));`

Just because whoops and yay is what we use everywhere already.

Reply:
No problem.  Please look at my validation though. I like to validate the long and dirty way... because I'm paranoid.  Laravel validation seems too easy.... 

About lang issues... we need to export to lang files in long run.....

(We discussed the json error thing at one point).